packages/dd-trace/src/tracer.js

```diff
@@ -122,6 +122,33 @@ class DatadogTracer extends Tracer {
 <meta name="dd-trace-id" content="${traceId}" />\
 <meta name="dd-trace-time" content="${traceTime}" />`
   }
+
+  /**
+   * Links an authenticated user to the current trace
+   * @param {object} user Properties of the authenticated user. Accepts custom fields
+   * @param {string} user.id Unique identifier of the user. Mandatory
+   * @param {string} [user.email] Email of the user
+   * @param {string} [user.name] User-friendly name of the user
+   * @param {string} [user.session_id] Session ID of the user
+   * @param {string} [user.role]
```

Comment:
Are these properties documented anywhere? How does a user know what to set these to?

Reply:
There is an RFC I can give you, and I assume (and will make sure) that there will be proper documentation for that in the doc website. I'll also documente it in API.md anyway.